javascript/chronometer.js

```diff
@@ -1,37 +1,60 @@
 class Chronometer {
   constructor() {
-    // ... your code goes here
+    this.currentTime = 0;
+    this.intervalId = null;
   }
 
   start(callback) {
-    // ... your code goes here
+   this.intervalId = setInterval(() => {
+     if(callback){
+       callback();
+     }
+     this.currentTime++
+   }, 1000)
   }
 
   getMinutes() {
-    // ... your code goes here
+    let minutes = this.currentTime/60;
+    minutes = Math.floor(minutes)
+   return minutes
```

Comment:
For shorter syntax you combine your lines like this:

return Math.floor(this.currentTime/60)